phpunit.multisite.xml

```diff
@@ -0,0 +1,24 @@
+<phpunit
```

Comment:
Core puts this file in `tests/phpunit/multisite.xml` but I thought it would be better to keep them side-by-side to ensure they're kept up-to-date. AFAIK there is no support for inheritance for these files so we need to duplicate them 😕